src/modules/core/components/Dialog/DialogSection.css

```diff
@@ -5,6 +5,9 @@
 }
 
 .themeHeading {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
```

Comment:
Please check if this change doesn't break the styles of the modals that don't a toggle.

Reply:
i was checking :)